dev-packages/node-integration-tests/utils/runner.ts

```diff
@@ -483,6 +483,7 @@ export function createRunner(...paths: string[]) {
           method: 'get' | 'post',
           path: string,
           headers: Record<string, string> = {},
+          data?: any, // axios accept any as data
```

Comment:
Do we actually need these changes now? I don't think so, unless I am missing something? :) If so, I'd just not do this change, no need to add stuff we don't need (and later nobody knows anymore if this is used somewhere etc.)

Reply:
well I needed them as first I was using post request and in this way you cannot sent the body as it was missing. The post request for axios expects body as 2nd param, but here you had headers which was wrong anyway - for post requests options like headers are passed as 3rd param, so imho this was bug anyway. I would leave it then